cep-recipe-extensions.md

```diff
@@ -0,0 +1,54 @@
+# Extensions to the recipe spec
+
+This document describes extensions to the recipe specification as merged in cep-14.
+
+## The `build.post_process` section
+
+We became aware that the `conda_build_config.yaml` has ways to specify post-processing steps. These are regex replacements that are performed on any new files. 
```

Comment:
```suggestion
The `conda_build_config.yaml` configuration file has ways to specify post-processing steps. These are regex replacements that are performed on any new files. 
```

Link to documentation, if available.

Reply:
I don't think that there are any docs for this 🙈